src/components/with-data-services/redux/saga.ts

```diff
@@ -12,15 +12,15 @@ import type { DataService } from '../../../types';
 
 function* getDataServicesRequested({
   payload: {
-    params: { dataseturi, size }
+    params: { dataseturi, size, endpointDescription }
   }
 }: ReturnType<typeof actions.getDataServicesRequested>) {
-  if (!dataseturi) {
+  if (!dataseturi && !endpointDescription) {
```

Comment:
er det kanskje || eller misforstår jeg noe?

Reply:
Trenger ikke datasetturi for den spørringen jeg skal gjøre, men så at noe kode du (?) hadde skrevet trengte den conditionen? Skrev det sånn slik at den ikke skulle trenge å endres, men fortsatt funke for mitt bruk. Med || hadde ikke min case funket.